src/main/convert_expressions.c

```diff
@@ -535,6 +535,26 @@ get_exp_val_from_pyval(AerospikeClient *self, as_static_pool *static_pool,
             *new_entry = tmp_entry;
         }
     }
+    else if (!strcmp(py_obj->ob_type->tp_name, "aerospike.KeyOrderedDict")) {
```

Comment:
I'm not sure why we needed this structure in the python client - maybe it used the as_orderedmap?

But I think this should also work if you use an [OrderedDict](https://docs.python.org/3/library/collections.html#ordereddict-objects)

Reply:
Hi Kevin, I'll use this for now since I don't want to make any breaking changes to the API